hledger-lib/Hledger/Data/Journal.hs

```diff
@@ -75,6 +75,7 @@ import Data.Array.ST
 import Data.Functor.Identity (Identity(..))
 import qualified Data.HashTable.ST.Cuckoo as HT
 import Data.List
+import Data.List.Extra(groupSortBy)
```

Comment:
Small suggestion: you may want to put space after module name `import Data.List.Extrta␣(groupSortBy)`